logicrunner/goplugin/preprocessor/main.go

```diff
@@ -436,6 +436,8 @@ func generateZeroListOfTypes(parsed *parsedFile, name string, list *ast.FieldLis
 			initializer = "bool(false)"
 		case "string":
 			initializer = `""`
+		case "foundation.Reference":
```

Comment:
In this task we should also start using core.RecordRef as a Reference. And set better name for  core.RecordRef -> core.Reference as it will be used in multiple levels, not only storage